CellFabric/Cell_Fabric_FinFET__Mock/canvas.py

```diff
@@ -31,6 +31,7 @@ def __init__( self, fin_u, fin, finDummy, gate, gateDummy,
         activeOffset = activeWidth//2 + finDummy*p['Fin']['Pitch']-p['Fin']['Pitch']//2
         activePitch = self.unitCellHeight
         RVTWidth = activeWidth1 + 2*p['Feol']['active_enclosure']
+        ##PcLength = (gate-1)*p['Poly']['Pitch']+p['Poly']['Width']+2*p['Feol']['PcExt']
   
         self.m0 = self.addGen( Wire( 'm0', 'M0', 'v',
```

Comment:
Does LISD replace m0? If so, we should probably remove m0 from both canvas & PDK Abstraction (it never gets used)

Reply:
Yes, I will remove M0.

Reply:
Thank you !

Reply:
@arvuce22 are you almost done with this? Would like to get this pull request merged as soon as possible. We can deal with this as part of a separate pull request if needed.

Let me know if you need any help merging into master. I can take over that part if needed. I want to start working on the device pin annotation.

Reply:
Item has been addressed. Marking as resolved.